### ISSUE TITLE
Simple 3D segmentation with U-Net example

### ISSUE THREAD

**Issue by bilwaj** [NifTK/NiftyNet#14]
Hi,
First, thanks for a nice package. I had no problems running some of the pre-trained BRATS models.

However, when I was trying to use net_segment.py to train using some of my own data, I ran into roadblocks. 

I get something to the effect of 
sh$ net_segment -c train_myconfig.ini

NiftyNet version 0.2.0.post1.dev481
usage: net_segment [-h] [--cuda_devices] [--num_threads] [--num_gpus]
                   [--model_dir]
                   {train,inference}
net_segment: error: too few arguments

train_myconfig.ini  is directly copied from:
https://github.com/NifTK/NiftyNet/blob/dev/config/unet_config.ini
and filepaths are updated to reflect my local system. I am on a MacBook OSX10.12.6 (No GPU) : File contents pasted below

[T1]
path_to_search = /Users/bilwaj/Dropbox/DeepVertebra/train/img
filename_contains = img
filename_not_contains =
spatial_window_size = (3, 64, 64)
pixdim = (1.0, 1.0, 1.0)
axcodes=(A, R, S)
interp_order = 3

[parcellation]
path_to_search = /Users/bilwaj/Dropbox/DeepVertebra/train/mask
filename_contains = img
filename_not_contains =
spatial_window_size = (3, 64, 64)
pixdim = (1.0, 1.0, 1.0)
axcodes=(A, R, S)
interp_order = 0

[SYSTEM]
cuda_devices = ""
num_threads = 2
num_gpus = 1
model_dir = /Users/bilwaj/Dropbox/DeepVertebra/niftynet

[NETWORK]
name = unet
activation_function = prelu
batch_size = 1
decay = 0
reg_type = L2

############################### volume level preprocessing
volume_padding_size = 0
############################# histogram normalisation
histogram_ref_file = ./label_mapping.txt
norm_type = percentile
cutoff = (0.01, 0.99)
normalisation = True
whitening = True
normalise_foreground_only=True
foreground_type = otsu_plus
multimod_foreground_type = and

queue_length = 128
window_sampling = uniform

[TRAINING]
sample_per_volume = 32
rotation_angle = (-10.0, 10.0)
scaling_percentage = (-10.0, 10.0)
lr = 0.01
loss_type = Dice
starting_iter = 0
save_every_n = 5
max_iter = 10
max_checkpoints = 20

[INFERENCE]
border = (0, 0, 0)
#inference_iter = 10
save_seg_dir = ./output/unet
output_interp_order = 0
spatial_window_size = (3, 64, 64)

############################ custom configuration sections
[SEGMENTATION]
image = T1
label = parcellation
output_prob = False
num_classes = 2
label_normalisation = True

**Comment by bilwaj** [comment]
sh$ net_segment -c train_myconfig.ini train 
seems to fix this particular issue

**Comment by wyli** [comment]
Many thanks for the feedback!
